src/components/header/Header.module.css

```diff
@@ -44,8 +41,6 @@
   color: #fff;
   text-decoration: none;
   transition: all .15s cubic-bezier(0.52, 0.51, 0.39, 0.99);
```

Comment:
could we use yellow btn on mobile too?
<img width="400" alt="Screenshot 2023-04-03 at 22 05 16" src="https://user-images.githubusercontent.com/4715396/229628300-44fffc37-73a2-4231-8515-5bd05c80c92b.png">